Add namespace field to EtcdReplicators (#1940)
sensu/sensu-docs: content/sensu-go/5.15/reference/etcdreplicators.md

metadata     |      |
-------------|------

> description  | Top-level scope that contains the replicator `name`. Namespace is not supported, as EtcdReplicators are cluster-wide resources.

Reviewer: ```suggestion
description  | Top-level scope that contains the replicator `name`. Namespace is not supported because EtcdReplicators are cluster-wide resources.
```

> description  | Top-level scope that contains the replicator `name`. Namespace is not supported because EtcdReplicators are cluster-wide resources.

Reviewer: ```suggestion
description  | Top-level scope that contains the replicator `name`. Namespace is not supported in the metadata because EtcdReplicators are cluster-wide resources.
```

type         | String
example      | {{< highlight shell >}}resource: Role{{< /highlight >}}

namespace    |      |
-------------|-------

> description  | Namespace to constrain replication to. If not supplied, all namespaces for a given resource are replicated.

Reviewer: ```suggestion
description  | Namespace to constrain replication to. If you do not include `namespace`, all namespaces for a given resource are replicated.
```

If you replicate the following four examples for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources, you can expect a full replication of [RBAC policy][3].

> _**NOTE**: If you do not specify a [namespace][2] when you create a replicator, all namespaces for a given resource are replicated._

Reviewer: ```suggestion
_**NOTE**: If you do not specify a namespace when you create a replicator, all namespaces for a given resource are replicated._
```
